src/test/unittest/unittest.ps1

```diff
@@ -1003,6 +1003,19 @@ function require_short_path {
     }
 }
 
+function init_ctl_config_env() {
+	$Env:PMEMBLK_CONF="fallc.at_create=0"
+	$Env:PMEMOBJ_CONF="fallc.at_create=0"
+	$Env:PMEMLOG_CONF="fallc.at_create=0"
+}
+
+function set_ctl_config() {
```

Comment:
This function name tells me nothing about why it's needed.
`fallocate_at_create_disable`
?